{"uid":"738b21c84e069566","name":"An RTP is sent through POSTE API","fullName":"tests.test_debtor_service_provider_availability#test_send_rtp_to_poste","historyId":"9ee43ed2ff7f9ef504e1b4a4edc013a4","time":{"start":1750921680589,"stop":1750921681034,"duration":445},"status":"broken","statusMessage":"requests.exceptions.SSLError: HTTPSConnectionPool(host='rtp.svil.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)')))","statusTrace":"self = <urllib3.connectionpool.HTTPSConnectionPool object at 0x7f386282d480>\nconn = <urllib3.connection.HTTPSConnection object at 0x7f3862185090>\nmethod = 'POST', url = '/iso20022/api/v1/sepa-request-to-pay-requests'\nbody = b'{\"resourceId\": \"3987fdd4-a77d-47df-b714-fec4b84d2338\", \"Document\": {\"CdtrPmtActvtnReq\": {\"GrpHdr\": {\"MsgId\": \"3987fd...WyVEjKB3H3YOj9jG90Qb2q\"]}, \"NclsdFile\": []}]}]}}, \"callbackUrl\": \"https://api-rtp-cb.uat.cstar.pagopa.it/rtp/cb/send\"}'\nheaders = {'User-Agent': 'python-requests/2.32.4', 'Accept-Encoding': 'gzip, deflate', 'Accept': 'application/json', 'Connection...987fdd4-a77d-47df-b714-fec4b84d2338', 'X-Request-ID': 'd2648e34-0cff-44cf-be76-a342a9ca4c26', 'Content-Length': '1477'}\nretries = Retry(total=0, connect=None, read=False, redirect=None, status=None)\ntimeout = Timeout(connect=5000, read=5000, total=None), chunked = False\nresponse_conn = <urllib3.connection.HTTPSConnection object at 0x7f3862185090>\npreload_content = False, decode_content = False, enforce_content_length = True\n\n    def _make_request(\n        self,\n        conn: BaseHTTPConnection,\n        method: str,\n        url: str,\n        body: _TYPE_BODY | None = None,\n        headers: typing.Mapping[str, str] | None = None,\n        retries: Retry | None = None,\n        timeout: _TYPE_TIMEOUT = _DEFAULT_TIMEOUT,\n        chunked: bool = False,\n        response_conn: BaseHTTPConnection | None = None,\n        preload_content: bool = True,\n        decode_content: bool = True,\n        enforce_content_length: bool = True,\n    ) -> BaseHTTPResponse:\n        \"\"\"\n        Perform a request on a given urllib connection object taken from our\n        pool.\n    \n        :param conn:\n            a connection from one of our connection pools\n    \n        :param method:\n            HTTP request method (such as GET, POST, PUT, etc.)\n    \n        :param url:\n            The URL to perform the request on.\n    \n        :param body:\n            Data to send in the request body, either :class:`str`, :class:`bytes`,\n            an iterable of :class:`str`/:class:`bytes`, or a file-like object.\n    \n        :param headers:\n            Dictionary of custom headers to send, such as User-Agent,\n            If-None-Match, etc. If None, pool headers are used. If provided,\n            these headers completely replace any pool-specific headers.\n    \n        :param retries:\n            Configure the number of retries to allow before raising a\n            :class:`~urllib3.exceptions.MaxRetryError` exception.\n    \n            Pass ``None`` to retry until you receive a response. Pass a\n            :class:`~urllib3.util.retry.Retry` object for fine-grained control\n            over different types of retries.\n            Pass an integer number to retry connection errors that many times,\n            but no other types of errors. Pass zero to never retry.\n    \n            If ``False``, then retries are disabled and any exception is raised\n            immediately. Also, instead of raising a MaxRetryError on redirects,\n            the redirect response will be returned.\n    \n        :type retries: :class:`~urllib3.util.retry.Retry`, False, or an int.\n    \n        :param timeout:\n            If specified, overrides the default timeout for this one\n            request. It may be a float (in seconds) or an instance of\n            :class:`urllib3.util.Timeout`.\n    \n        :param chunked:\n            If True, urllib3 will send the body using chunked transfer\n            encoding. Otherwise, urllib3 will send the body using the standard\n            content-length form. Defaults to False.\n    \n        :param response_conn:\n            Set this to ``None`` if you will handle releasing the connection or\n            set the connection to have the response release it.\n    \n        :param preload_content:\n          If True, the response's body will be preloaded during construction.\n    \n        :param decode_content:\n            If True, will attempt to decode the body based on the\n            'content-encoding' header.\n    \n        :param enforce_content_length:\n            Enforce content length checking. Body returned by server must match\n            value of Content-Length header, if present. Otherwise, raise error.\n        \"\"\"\n        self.num_requests += 1\n    \n        timeout_obj = self._get_timeout(timeout)\n        timeout_obj.start_connect()\n        conn.timeout = Timeout.resolve_default_timeout(timeout_obj.connect_timeout)\n    \n        try:\n            # Trigger any extra validation we need to do.\n            try:\n>               self._validate_conn(conn)\n\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/urllib3/connectionpool.py:464: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/urllib3/connectionpool.py:1093: in _validate_conn\n    conn.connect()\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/urllib3/connection.py:790: in connect\n    sock_and_verified = _ssl_wrap_socket_and_match_hostname(\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/urllib3/connection.py:969: in _ssl_wrap_socket_and_match_hostname\n    ssl_sock = ssl_wrap_socket(\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/urllib3/util/ssl_.py:480: in ssl_wrap_socket\n    ssl_sock = _ssl_wrap_socket_impl(sock, context, tls_in_tls, server_hostname)\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/urllib3/util/ssl_.py:524: in _ssl_wrap_socket_impl\n    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/ssl.py:455: in wrap_socket\n    return self.sslsocket_class._create(\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/ssl.py:1076: in _create\n    self.do_handshake()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <ssl.SSLSocket [closed] fd=-1, family=2, type=1, proto=6>, block = False\n\n    @_sslcopydoc\n    def do_handshake(self, block=False):\n        self._check_connected()\n        timeout = self.gettimeout()\n        try:\n            if timeout == 0.0 and block:\n                self.settimeout(None)\n>           self._sslobj.do_handshake()\nE           ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)\n\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/ssl.py:1372: SSLCertVerificationError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <urllib3.connectionpool.HTTPSConnectionPool object at 0x7f386282d480>\nmethod = 'POST', url = '/iso20022/api/v1/sepa-request-to-pay-requests'\nbody = b'{\"resourceId\": \"3987fdd4-a77d-47df-b714-fec4b84d2338\", \"Document\": {\"CdtrPmtActvtnReq\": {\"GrpHdr\": {\"MsgId\": \"3987fd...WyVEjKB3H3YOj9jG90Qb2q\"]}, \"NclsdFile\": []}]}]}}, \"callbackUrl\": \"https://api-rtp-cb.uat.cstar.pagopa.it/rtp/cb/send\"}'\nheaders = {'User-Agent': 'python-requests/2.32.4', 'Accept-Encoding': 'gzip, deflate', 'Accept': 'application/json', 'Connection...987fdd4-a77d-47df-b714-fec4b84d2338', 'X-Request-ID': 'd2648e34-0cff-44cf-be76-a342a9ca4c26', 'Content-Length': '1477'}\nretries = Retry(total=0, connect=None, read=False, redirect=None, status=None)\nredirect = False, assert_same_host = False\ntimeout = Timeout(connect=5000, read=5000, total=None), pool_timeout = None\nrelease_conn = False, chunked = False, body_pos = None, preload_content = False\ndecode_content = False, response_kw = {}\nparsed_url = Url(scheme=None, auth=None, host=None, port=None, path='/iso20022/api/v1/sepa-request-to-pay-requests', query=None, fragment=None)\ndestination_scheme = None, conn = None, release_this_conn = True\nhttp_tunnel_required = False, err = None, clean_exit = False\n\n    def urlopen(  # type: ignore[override]\n        self,\n        method: str,\n        url: str,\n        body: _TYPE_BODY | None = None,\n        headers: typing.Mapping[str, str] | None = None,\n        retries: Retry | bool | int | None = None,\n        redirect: bool = True,\n        assert_same_host: bool = True,\n        timeout: _TYPE_TIMEOUT = _DEFAULT_TIMEOUT,\n        pool_timeout: int | None = None,\n        release_conn: bool | None = None,\n        chunked: bool = False,\n        body_pos: _TYPE_BODY_POSITION | None = None,\n        preload_content: bool = True,\n        decode_content: bool = True,\n        **response_kw: typing.Any,\n    ) -> BaseHTTPResponse:\n        \"\"\"\n        Get a connection from the pool and perform an HTTP request. This is the\n        lowest level call for making a request, so you'll need to specify all\n        the raw details.\n    \n        .. note::\n    \n           More commonly, it's appropriate to use a convenience method\n           such as :meth:`request`.\n    \n        .. note::\n    \n           `release_conn` will only behave as expected if\n           `preload_content=False` because we want to make\n           `preload_content=False` the default behaviour someday soon without\n           breaking backwards compatibility.\n    \n        :param method:\n            HTTP request method (such as GET, POST, PUT, etc.)\n    \n        :param url:\n            The URL to perform the request on.\n    \n        :param body:\n            Data to send in the request body, either :class:`str`, :class:`bytes`,\n            an iterable of :class:`str`/:class:`bytes`, or a file-like object.\n    \n        :param headers:\n            Dictionary of custom headers to send, such as User-Agent,\n            If-None-Match, etc. If None, pool headers are used. If provided,\n            these headers completely replace any pool-specific headers.\n    \n        :param retries:\n            Configure the number of retries to allow before raising a\n            :class:`~urllib3.exceptions.MaxRetryError` exception.\n    \n            If ``None`` (default) will retry 3 times, see ``Retry.DEFAULT``. Pass a\n            :class:`~urllib3.util.retry.Retry` object for fine-grained control\n            over different types of retries.\n            Pass an integer number to retry connection errors that many times,\n            but no other types of errors. Pass zero to never retry.\n    \n            If ``False``, then retries are disabled and any exception is raised\n            immediately. Also, instead of raising a MaxRetryError on redirects,\n            the redirect response will be returned.\n    \n        :type retries: :class:`~urllib3.util.retry.Retry`, False, or an int.\n    \n        :param redirect:\n            If True, automatically handle redirects (status codes 301, 302,\n            303, 307, 308). Each redirect counts as a retry. Disabling retries\n            will disable redirect, too.\n    \n        :param assert_same_host:\n            If ``True``, will make sure that the host of the pool requests is\n            consistent else will raise HostChangedError. When ``False``, you can\n            use the pool on an HTTP proxy and request foreign hosts.\n    \n        :param timeout:\n            If specified, overrides the default timeout for this one\n            request. It may be a float (in seconds) or an instance of\n            :class:`urllib3.util.Timeout`.\n    \n        :param pool_timeout:\n            If set and the pool is set to block=True, then this method will\n            block for ``pool_timeout`` seconds and raise EmptyPoolError if no\n            connection is available within the time period.\n    \n        :param bool preload_content:\n            If True, the response's body will be preloaded into memory.\n    \n        :param bool decode_content:\n            If True, will attempt to decode the body based on the\n            'content-encoding' header.\n    \n        :param release_conn:\n            If False, then the urlopen call will not release the connection\n            back into the pool once a response is received (but will release if\n            you read the entire contents of the response such as when\n            `preload_content=True`). This is useful if you're not preloading\n            the response's content immediately. You will need to call\n            ``r.release_conn()`` on the response ``r`` to return the connection\n            back into the pool. If None, it takes the value of ``preload_content``\n            which defaults to ``True``.\n    \n        :param bool chunked:\n            If True, urllib3 will send the body using chunked transfer\n            encoding. Otherwise, urllib3 will send the body using the standard\n            content-length form. Defaults to False.\n    \n        :param int body_pos:\n            Position to seek to in file-like body in the event of a retry or\n            redirect. Typically this won't need to be set because urllib3 will\n            auto-populate the value when needed.\n        \"\"\"\n        parsed_url = parse_url(url)\n        destination_scheme = parsed_url.scheme\n    \n        if headers is None:\n            headers = self.headers\n    \n        if not isinstance(retries, Retry):\n            retries = Retry.from_int(retries, redirect=redirect, default=self.retries)\n    \n        if release_conn is None:\n            release_conn = preload_content\n    \n        # Check host\n        if assert_same_host and not self.is_same_host(url):\n            raise HostChangedError(self, url, retries)\n    \n        # Ensure that the URL we're connecting to is properly encoded\n        if url.startswith(\"/\"):\n            url = to_str(_encode_target(url))\n        else:\n            url = to_str(parsed_url.url)\n    \n        conn = None\n    \n        # Track whether `conn` needs to be released before\n        # returning/raising/recursing. Update this variable if necessary, and\n        # leave `release_conn` constant throughout the function. That way, if\n        # the function recurses, the original value of `release_conn` will be\n        # passed down into the recursive call, and its value will be respected.\n        #\n        # See issue #651 [1] for details.\n        #\n        # [1] <https://github.com/urllib3/urllib3/issues/651>\n        release_this_conn = release_conn\n    \n        http_tunnel_required = connection_requires_http_tunnel(\n            self.proxy, self.proxy_config, destination_scheme\n        )\n    \n        # Merge the proxy headers. Only done when not using HTTP CONNECT. We\n        # have to copy the headers dict so we can safely change it without those\n        # changes being reflected in anyone else's copy.\n        if not http_tunnel_required:\n            headers = headers.copy()  # type: ignore[attr-defined]\n            headers.update(self.proxy_headers)  # type: ignore[union-attr]\n    \n        # Must keep the exception bound to a separate variable or else Python 3\n        # complains about UnboundLocalError.\n        err = None\n    \n        # Keep track of whether we cleanly exited the except block. This\n        # ensures we do proper cleanup in finally.\n        clean_exit = False\n    \n        # Rewind body position, if needed. Record current position\n        # for future rewinds in the event of a redirect/retry.\n        body_pos = set_file_position(body, body_pos)\n    \n        try:\n            # Request a connection from the queue.\n            timeout_obj = self._get_timeout(timeout)\n            conn = self._get_conn(timeout=pool_timeout)\n    \n            conn.timeout = timeout_obj.connect_timeout  # type: ignore[assignment]\n    \n            # Is this a closed/new connection that requires CONNECT tunnelling?\n            if self.proxy is not None and http_tunnel_required and conn.is_closed:\n                try:\n                    self._prepare_proxy(conn)\n                except (BaseSSLError, OSError, SocketTimeout) as e:\n                    self._raise_timeout(\n                        err=e, url=self.proxy.url, timeout_value=conn.timeout\n                    )\n                    raise\n    \n            # If we're going to release the connection in ``finally:``, then\n            # the response doesn't need to know about the connection. Otherwise\n            # it will also try to release it and we'll have a double-release\n            # mess.\n            response_conn = conn if not release_conn else None\n    \n            # Make the request on the HTTPConnection object\n>           response = self._make_request(\n                conn,\n                method,\n                url,\n                timeout=timeout_obj,\n                body=body,\n                headers=headers,\n                chunked=chunked,\n                retries=retries,\n                response_conn=response_conn,\n                preload_content=preload_content,\n                decode_content=decode_content,\n                **response_kw,\n            )\n\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/urllib3/connectionpool.py:787: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <urllib3.connectionpool.HTTPSConnectionPool object at 0x7f386282d480>\nconn = <urllib3.connection.HTTPSConnection object at 0x7f3862185090>\nmethod = 'POST', url = '/iso20022/api/v1/sepa-request-to-pay-requests'\nbody = b'{\"resourceId\": \"3987fdd4-a77d-47df-b714-fec4b84d2338\", \"Document\": {\"CdtrPmtActvtnReq\": {\"GrpHdr\": {\"MsgId\": \"3987fd...WyVEjKB3H3YOj9jG90Qb2q\"]}, \"NclsdFile\": []}]}]}}, \"callbackUrl\": \"https://api-rtp-cb.uat.cstar.pagopa.it/rtp/cb/send\"}'\nheaders = {'User-Agent': 'python-requests/2.32.4', 'Accept-Encoding': 'gzip, deflate', 'Accept': 'application/json', 'Connection...987fdd4-a77d-47df-b714-fec4b84d2338', 'X-Request-ID': 'd2648e34-0cff-44cf-be76-a342a9ca4c26', 'Content-Length': '1477'}\nretries = Retry(total=0, connect=None, read=False, redirect=None, status=None)\ntimeout = Timeout(connect=5000, read=5000, total=None), chunked = False\nresponse_conn = <urllib3.connection.HTTPSConnection object at 0x7f3862185090>\npreload_content = False, decode_content = False, enforce_content_length = True\n\n    def _make_request(\n        self,\n        conn: BaseHTTPConnection,\n        method: str,\n        url: str,\n        body: _TYPE_BODY | None = None,\n        headers: typing.Mapping[str, str] | None = None,\n        retries: Retry | None = None,\n        timeout: _TYPE_TIMEOUT = _DEFAULT_TIMEOUT,\n        chunked: bool = False,\n        response_conn: BaseHTTPConnection | None = None,\n        preload_content: bool = True,\n        decode_content: bool = True,\n        enforce_content_length: bool = True,\n    ) -> BaseHTTPResponse:\n        \"\"\"\n        Perform a request on a given urllib connection object taken from our\n        pool.\n    \n        :param conn:\n            a connection from one of our connection pools\n    \n        :param method:\n            HTTP request method (such as GET, POST, PUT, etc.)\n    \n        :param url:\n            The URL to perform the request on.\n    \n        :param body:\n            Data to send in the request body, either :class:`str`, :class:`bytes`,\n            an iterable of :class:`str`/:class:`bytes`, or a file-like object.\n    \n        :param headers:\n            Dictionary of custom headers to send, such as User-Agent,\n            If-None-Match, etc. If None, pool headers are used. If provided,\n            these headers completely replace any pool-specific headers.\n    \n        :param retries:\n            Configure the number of retries to allow before raising a\n            :class:`~urllib3.exceptions.MaxRetryError` exception.\n    \n            Pass ``None`` to retry until you receive a response. Pass a\n            :class:`~urllib3.util.retry.Retry` object for fine-grained control\n            over different types of retries.\n            Pass an integer number to retry connection errors that many times,\n            but no other types of errors. Pass zero to never retry.\n    \n            If ``False``, then retries are disabled and any exception is raised\n            immediately. Also, instead of raising a MaxRetryError on redirects,\n            the redirect response will be returned.\n    \n        :type retries: :class:`~urllib3.util.retry.Retry`, False, or an int.\n    \n        :param timeout:\n            If specified, overrides the default timeout for this one\n            request. It may be a float (in seconds) or an instance of\n            :class:`urllib3.util.Timeout`.\n    \n        :param chunked:\n            If True, urllib3 will send the body using chunked transfer\n            encoding. Otherwise, urllib3 will send the body using the standard\n            content-length form. Defaults to False.\n    \n        :param response_conn:\n            Set this to ``None`` if you will handle releasing the connection or\n            set the connection to have the response release it.\n    \n        :param preload_content:\n          If True, the response's body will be preloaded during construction.\n    \n        :param decode_content:\n            If True, will attempt to decode the body based on the\n            'content-encoding' header.\n    \n        :param enforce_content_length:\n            Enforce content length checking. Body returned by server must match\n            value of Content-Length header, if present. Otherwise, raise error.\n        \"\"\"\n        self.num_requests += 1\n    \n        timeout_obj = self._get_timeout(timeout)\n        timeout_obj.start_connect()\n        conn.timeout = Timeout.resolve_default_timeout(timeout_obj.connect_timeout)\n    \n        try:\n            # Trigger any extra validation we need to do.\n            try:\n                self._validate_conn(conn)\n            except (SocketTimeout, BaseSSLError) as e:\n                self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)\n                raise\n    \n        # _validate_conn() starts the connection to an HTTPS proxy\n        # so we need to wrap errors with 'ProxyError' here too.\n        except (\n            OSError,\n            NewConnectionError,\n            TimeoutError,\n            BaseSSLError,\n            CertificateError,\n            SSLError,\n        ) as e:\n            new_e: Exception = e\n            if isinstance(e, (BaseSSLError, CertificateError)):\n                new_e = SSLError(e)\n            # If the connection didn't successfully connect to it's proxy\n            # then there\n            if isinstance(\n                new_e, (OSError, NewConnectionError, TimeoutError, SSLError)\n            ) and (conn and conn.proxy and not conn.has_connected_to_proxy):\n                new_e = _wrap_proxy_error(new_e, conn.proxy.scheme)\n>           raise new_e\nE           urllib3.exceptions.SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)\n\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/urllib3/connectionpool.py:488: SSLError\n\nThe above exception was the direct cause of the following exception:\n\nself = <requests.adapters.HTTPAdapter object at 0x7f3862260650>\nrequest = <PreparedRequest [POST]>, stream = False\ntimeout = Timeout(connect=5000, read=5000, total=None), verify = True\ncert = None, proxies = OrderedDict()\n\n    def send(\n        self, request, stream=False, timeout=None, verify=True, cert=None, proxies=None\n    ):\n        \"\"\"Sends PreparedRequest object. Returns Response object.\n    \n        :param request: The :class:`PreparedRequest <PreparedRequest>` being sent.\n        :param stream: (optional) Whether to stream the request content.\n        :param timeout: (optional) How long to wait for the server to send\n            data before giving up, as a float, or a :ref:`(connect timeout,\n            read timeout) <timeouts>` tuple.\n        :type timeout: float or tuple or urllib3 Timeout object\n        :param verify: (optional) Either a boolean, in which case it controls whether\n            we verify the server's TLS certificate, or a string, in which case it\n            must be a path to a CA bundle to use\n        :param cert: (optional) Any user-provided SSL certificate to be trusted.\n        :param proxies: (optional) The proxies dictionary to apply to the request.\n        :rtype: requests.Response\n        \"\"\"\n    \n        try:\n            conn = self.get_connection_with_tls_context(\n                request, verify, proxies=proxies, cert=cert\n            )\n        except LocationValueError as e:\n            raise InvalidURL(e, request=request)\n    \n        self.cert_verify(conn, request.url, verify, cert)\n        url = self.request_url(request, proxies)\n        self.add_headers(\n            request,\n            stream=stream,\n            timeout=timeout,\n            verify=verify,\n            cert=cert,\n            proxies=proxies,\n        )\n    \n        chunked = not (request.body is None or \"Content-Length\" in request.headers)\n    \n        if isinstance(timeout, tuple):\n            try:\n                connect, read = timeout\n                timeout = TimeoutSauce(connect=connect, read=read)\n            except ValueError:\n                raise ValueError(\n                    f\"Invalid timeout {timeout}. Pass a (connect, read) timeout tuple, \"\n                    f\"or a single float to set both timeouts to the same value.\"\n                )\n        elif isinstance(timeout, TimeoutSauce):\n            pass\n        else:\n            timeout = TimeoutSauce(connect=timeout, read=timeout)\n    \n        try:\n>           resp = conn.urlopen(\n                method=request.method,\n                url=url,\n                body=request.body,\n                headers=request.headers,\n                redirect=False,\n                assert_same_host=False,\n                preload_content=False,\n                decode_content=False,\n                retries=self.max_retries,\n                timeout=timeout,\n                chunked=chunked,\n            )\n\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/requests/adapters.py:667: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/urllib3/connectionpool.py:841: in urlopen\n    retries = retries.increment(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = Retry(total=0, connect=None, read=False, redirect=None, status=None)\nmethod = 'POST', url = '/iso20022/api/v1/sepa-request-to-pay-requests'\nresponse = None\nerror = SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)'))\n_pool = <urllib3.connectionpool.HTTPSConnectionPool object at 0x7f386282d480>\n_stacktrace = <traceback object at 0x7f38621319c0>\n\n    def increment(\n        self,\n        method: str | None = None,\n        url: str | None = None,\n        response: BaseHTTPResponse | None = None,\n        error: Exception | None = None,\n        _pool: ConnectionPool | None = None,\n        _stacktrace: TracebackType | None = None,\n    ) -> Self:\n        \"\"\"Return a new Retry object with incremented retry counters.\n    \n        :param response: A response object, or None, if the server did not\n            return a response.\n        :type response: :class:`~urllib3.response.BaseHTTPResponse`\n        :param Exception error: An error encountered during the request, or\n            None if the response was received successfully.\n    \n        :return: A new ``Retry`` object.\n        \"\"\"\n        if self.total is False and error:\n            # Disabled, indicate to re-raise the error.\n            raise reraise(type(error), error, _stacktrace)\n    \n        total = self.total\n        if total is not None:\n            total -= 1\n    \n        connect = self.connect\n        read = self.read\n        redirect = self.redirect\n        status_count = self.status\n        other = self.other\n        cause = \"unknown\"\n        status = None\n        redirect_location = None\n    \n        if error and self._is_connection_error(error):\n            # Connect retry?\n            if connect is False:\n                raise reraise(type(error), error, _stacktrace)\n            elif connect is not None:\n                connect -= 1\n    \n        elif error and self._is_read_error(error):\n            # Read retry?\n            if read is False or method is None or not self._is_method_retryable(method):\n                raise reraise(type(error), error, _stacktrace)\n            elif read is not None:\n                read -= 1\n    \n        elif error:\n            # Other retry?\n            if other is not None:\n                other -= 1\n    \n        elif response and response.get_redirect_location():\n            # Redirect retry?\n            if redirect is not None:\n                redirect -= 1\n            cause = \"too many redirects\"\n            response_redirect_location = response.get_redirect_location()\n            if response_redirect_location:\n                redirect_location = response_redirect_location\n            status = response.status\n    \n        else:\n            # Incrementing because of a server error like a 500 in\n            # status_forcelist and the given method is in the allowed_methods\n            cause = ResponseError.GENERIC_ERROR\n            if response and response.status:\n                if status_count is not None:\n                    status_count -= 1\n                cause = ResponseError.SPECIFIC_ERROR.format(status_code=response.status)\n                status = response.status\n    \n        history = self.history + (\n            RequestHistory(method, url, error, status, redirect_location),\n        )\n    \n        new_retry = self.new(\n            total=total,\n            connect=connect,\n            read=read,\n            redirect=redirect,\n            status=status_count,\n            other=other,\n            history=history,\n        )\n    \n        if new_retry.is_exhausted():\n            reason = error or ResponseError(cause)\n>           raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]\nE           urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='rtp.svil.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)')))\n\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/urllib3/util/retry.py:519: MaxRetryError\n\nDuring handling of the above exception, another exception occurred:\n\n    @allure.feature('RTP Send')\n    @allure.story('Service provider sends an RTP to POSTE directly')\n    @allure.title('An RTP is sent through POSTE API')\n    @pytest.mark.send\n    @pytest.mark.happy_path\n    @pytest.mark.poste\n    def test_send_rtp_to_poste():\n        rtp_data = generate_rtp_data()\n        cbi_payload = generate_cbi_rtp_data(rtp_data)\n    \n>       response = send_srtp_to_poste(cbi_payload)\n\nfunctional-tests/tests/test_debtor_service_provider_availability.py:121: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \napi/debtor_service_provider.py:25: in send_srtp_to_poste\n    return requests.post(\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/requests/api.py:115: in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/requests/api.py:59: in request\n    return session.request(method=method, url=url, **kwargs)\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/requests/sessions.py:589: in request\n    resp = self.send(prep, **send_kwargs)\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/requests/sessions.py:703: in send\n    r = adapter.send(request, **kwargs)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <requests.adapters.HTTPAdapter object at 0x7f3862260650>\nrequest = <PreparedRequest [POST]>, stream = False\ntimeout = Timeout(connect=5000, read=5000, total=None), verify = True\ncert = None, proxies = OrderedDict()\n\n    def send(\n        self, request, stream=False, timeout=None, verify=True, cert=None, proxies=None\n    ):\n        \"\"\"Sends PreparedRequest object. Returns Response object.\n    \n        :param request: The :class:`PreparedRequest <PreparedRequest>` being sent.\n        :param stream: (optional) Whether to stream the request content.\n        :param timeout: (optional) How long to wait for the server to send\n            data before giving up, as a float, or a :ref:`(connect timeout,\n            read timeout) <timeouts>` tuple.\n        :type timeout: float or tuple or urllib3 Timeout object\n        :param verify: (optional) Either a boolean, in which case it controls whether\n            we verify the server's TLS certificate, or a string, in which case it\n            must be a path to a CA bundle to use\n        :param cert: (optional) Any user-provided SSL certificate to be trusted.\n        :param proxies: (optional) The proxies dictionary to apply to the request.\n        :rtype: requests.Response\n        \"\"\"\n    \n        try:\n            conn = self.get_connection_with_tls_context(\n                request, verify, proxies=proxies, cert=cert\n            )\n        except LocationValueError as e:\n            raise InvalidURL(e, request=request)\n    \n        self.cert_verify(conn, request.url, verify, cert)\n        url = self.request_url(request, proxies)\n        self.add_headers(\n            request,\n            stream=stream,\n            timeout=timeout,\n            verify=verify,\n            cert=cert,\n            proxies=proxies,\n        )\n    \n        chunked = not (request.body is None or \"Content-Length\" in request.headers)\n    \n        if isinstance(timeout, tuple):\n            try:\n                connect, read = timeout\n                timeout = TimeoutSauce(connect=connect, read=read)\n            except ValueError:\n                raise ValueError(\n                    f\"Invalid timeout {timeout}. Pass a (connect, read) timeout tuple, \"\n                    f\"or a single float to set both timeouts to the same value.\"\n                )\n        elif isinstance(timeout, TimeoutSauce):\n            pass\n        else:\n            timeout = TimeoutSauce(connect=timeout, read=timeout)\n    \n        try:\n            resp = conn.urlopen(\n                method=request.method,\n                url=url,\n                body=request.body,\n                headers=request.headers,\n                redirect=False,\n                assert_same_host=False,\n                preload_content=False,\n                decode_content=False,\n                retries=self.max_retries,\n                timeout=timeout,\n                chunked=chunked,\n            )\n    \n        except (ProtocolError, OSError) as err:\n            raise ConnectionError(err, request=request)\n    \n        except MaxRetryError as e:\n            if isinstance(e.reason, ConnectTimeoutError):\n                # TODO: Remove this in 3.0.0: see #2811\n                if not isinstance(e.reason, NewConnectionError):\n                    raise ConnectTimeout(e, request=request)\n    \n            if isinstance(e.reason, ResponseError):\n                raise RetryError(e, request=request)\n    \n            if isinstance(e.reason, _ProxyError):\n                raise ProxyError(e, request=request)\n    \n            if isinstance(e.reason, _SSLError):\n                # This branch is for urllib3 v1.22 and later.\n>               raise SSLError(e, request=request)\nE               requests.exceptions.SSLError: HTTPSConnectionPool(host='rtp.svil.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)')))\n\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/requests/adapters.py:698: SSLError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1750921653791,"stop":1750921653848,"duration":57},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"story","value":"Service provider sends an RTP to POSTE directly"},{"name":"feature","value":"RTP Send"},{"name":"tag","value":"poste"},{"name":"tag","value":"happy_path"},{"name":"tag","value":"send"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_debtor_service_provider_availability"},{"name":"host","value":"fv-az1276-16"},{"name":"thread","value":"2431-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_debtor_service_provider_availability"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":57,"skipped":0,"passed":0,"unknown":0,"total":57},"items":[{"uid":"258a67a02043dc36","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/411//#testresult/258a67a02043dc36","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='rtp.svil.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)')))","time":{"start":1750856382363,"stop":1750856382752,"duration":389}},{"uid":"f3bccb7ad39d6285","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/408//#testresult/f3bccb7ad39d6285","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0557b9ae90>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750835302395,"stop":1750835437140,"duration":134745}},{"uid":"db0b91de8e344dd","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/407//#testresult/db0b91de8e344dd","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3e0039f4d0>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750780537389,"stop":1750780670777,"duration":133388}},{"uid":"808c7792ef20b62b","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/406//#testresult/808c7792ef20b62b","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0c6865ae90>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750779068183,"stop":1750779201593,"duration":133410}},{"uid":"1dc07602aba043b1","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/405//#testresult/1dc07602aba043b1","status":"broken","statusDetails":"OSError: Could not find the TLS certificate file, invalid path: /home/runner/work/rtp-platform-qa/rtp-platform-qa/config/cert.pem","time":{"start":1750777965692,"stop":1750777965693,"duration":1}},{"uid":"dc84809653bf1532","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/404//#testresult/dc84809653bf1532","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7895132e90>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750756725988,"stop":1750756860224,"duration":134236}},{"uid":"970bf40937eea695","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/403//#testresult/970bf40937eea695","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fd1c0c560d0>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750748875725,"stop":1750749009662,"duration":133937}},{"uid":"6f0f6b4e520793f6","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/402//#testresult/6f0f6b4e520793f6","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7e56c2e0d0>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750684871241,"stop":1750685007600,"duration":136359}},{"uid":"57a799f2d9f4bee9","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/401//#testresult/57a799f2d9f4bee9","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f340528f250>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750662510703,"stop":1750662646306,"duration":135603}},{"uid":"40885e7ced652fc9","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/400//#testresult/40885e7ced652fc9","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f920765a0d0>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750576084541,"stop":1750576220836,"duration":136295}},{"uid":"257cdf92bb30591c","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/399//#testresult/257cdf92bb30591c","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f99b804f250>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750489651935,"stop":1750489787142,"duration":135207}},{"uid":"bdc87548abc6f41b","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/398//#testresult/bdc87548abc6f41b","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f49405ef250>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750403245452,"stop":1750403381968,"duration":136516}},{"uid":"957d47383e84a297","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/397//#testresult/957d47383e84a297","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f4b15363250>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750335017992,"stop":1750335154023,"duration":136031}},{"uid":"b9d30daf6e270050","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/396//#testresult/b9d30daf6e270050","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fef93a720d0>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750329658096,"stop":1750329794563,"duration":136467}},{"uid":"99dc5542721f8c6","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/395//#testresult/99dc5542721f8c6","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdb8b523610>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750328594285,"stop":1750328728602,"duration":134317}},{"uid":"9138498fa233ff77","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/394//#testresult/9138498fa233ff77","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff1ed2f3250>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750317554072,"stop":1750317689390,"duration":135318}},{"uid":"76a40a53b6c77ef8","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/393//#testresult/76a40a53b6c77ef8","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fb0156e7610>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750316879510,"stop":1750317015794,"duration":136284}},{"uid":"60be80cdee5c3d4b","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/392//#testresult/60be80cdee5c3d4b","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f06a5323610>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750230495644,"stop":1750230630050,"duration":134406}},{"uid":"95818c65354b5941","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/391//#testresult/95818c65354b5941","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fc68a027610>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750170145671,"stop":1750170281368,"duration":135697}},{"uid":"d1847462de7055f7","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/389//#testresult/d1847462de7055f7","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1c28c21e50>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750144060530,"stop":1750144194619,"duration":134089}}]},"tags":["poste","send","happy_path"]},"source":"738b21c84e069566.json","parameterValues":[]}